src/command/mod.rs

```diff
@@ -0,0 +1,133 @@
+//! command definitions
+use error::Error;
```

Comment:
Please see [Style Guidelines](https://doc.rust-lang.org/1.0.0/style/style/imports.html). It recommends to seperate internal `use` and external `use`.